### PR TITLE
Fix docstrings

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Changes

- Addresses doc string issues.

# Known Issues

- We can add more examples to the docstring.

### CODE REVIEW

File: anomalib/core/callbacks/model_loader.py [hunk 1]
```diff
@@ -10,5 +10,8 @@ def __init__(self, weights_path):
         self.weights_path = weights_path
 
     def on_test_start(self, trainer, pl_module: LightningModule) -> None:  # pylint: disable=W0613
-        """Call when the test begins."""
+        """Call when the test begins.
+
+        Loads the model weights from into the PyTorch module.
```
Comment (djdameln) [inline anomalib/core/callbacks/model_loader.py:15]: Something's missing between `from` and `into`

File: anomalib/core/model/kde.py [hunk 2]
```diff
@@ -88,12 +93,12 @@ def fit(self, dataset: torch.Tensor) -> None:
...
     @staticmethod
-    def cov(tensor: torch.Tensor, bias: Optional[bool] = False) -> torch.Tensor:
+    def cov(tensor: Tensor, bias: Optional[bool] = False) -> Tensor:
         """Calculate covariance matrix.
 
         Args:
-            tensor: torch.Tensor: Input tensor from which covariance matrix is computed.
-            bias: Optional[bool]:  (Default value = False)
+            tensor (Tensor): Input tensor from which covariance matrix is computed.
+            bias (Optional[bool]):  (Default value = False)
```
Comment (djdameln) [inline anomalib/core/model/kde.py:101]: Can you add a description for this parameter?

File: anomalib/models/dfkde/normality_model.py [hunk 3]
```diff
@@ -78,15 +86,13 @@ def fit(self, dataset: torch.Tensor):
...
-    ) -> Tuple[torch.Tensor, torch.Tensor]:
+    def preprocess(self, feature_stack: Tensor, max_length: Optional[Tensor] = None) -> Tuple[Tensor, Tensor]:
         """Pre process the CNN features.
 
         Args:
           feature_stack: Features extracted from CNN
           max_length:
-          feature_stack: torch.Tensor:
+          feature_stack: Tensor:
           max_length: Optional[Tensor]:  (Default value = None)
```
Comment (djdameln) [inline anomalib/models/dfkde/normality_model.py:96]: Descriptions are missing here

**Comment by djdameln** [comment]
Few minor comments

File: anomalib/models/dfkde/normality_model.py (hunk 3 above)
Comment (ashwinvaidya17) [reply anomalib/models/dfkde/normality_model.py:96]: Thanks! I missed these.

File: anomalib/models/dfm/dfm_model.py [hunk 4]
```diff
@@ -39,13 +39,13 @@ def __init__(self):
...
 
-            Covariance matrix is not calculated directly using:
-                C = X.X^T
-            Instead, it is represented in terms of the Singular Value Decomposition of X:
-                X = U.S.V^T
-            Hence,
-                C = U.S^2.U^T
-            This simplifies the calculation of the log-likelihood without requiring full matrix inversion.
+        Covariance matrix is not calculated directly using:
+            C = X.X^T
```
Comment (samet-akcay) [inline anomalib/models/dfm/dfm_model.py:43]: maybe we could have these equations within quotes to properly render in the docs

Comment (samet-akcay) [reply anomalib/models/dfm/dfm_model.py:43]: like `C = X.X^T`?

File: anomalib/models/dfm/model.py [hunk 5]
```diff
@@ -78,8 +78,8 @@ def validation_step(self, batch, _):  # pylint: disable=arguments-differ
         Similar to the training step, features are extracted from the CNN for each batch.
 
         Args:
-          batch: Dict: Input batch
-          batch_idx: int: Index of the batch.
+          batch (List[Dict[str, Any]]): Input batch
+          _ (int): Index of the batch.
```
Comment (samet-akcay) [inline anomalib/models/dfm/model.py:82]: We could remove this. It'll appear in the documentation, which won't make sense to the reader.

**Review by samet-akcay — CHANGES_REQUESTED** [review]
only couple of minor comment

**Review by samet-akcay — APPROVED** [review]

**Review by djdameln — APPROVED** [review]

**Pull request merged**